src/ensRegistry.ts

```diff
@@ -139,8 +151,10 @@ export function handleNewResolver(event: NewResolverEvent): void {
 export function handleNewTTL(event: NewTTLEvent): void {
   let node = event.params.node.toHexString()
   let domain = getDomain(node)
-  domain.ttl = event.params.ttl
-  domain.save()
+  if(domain){
```

Comment:
If domain is null here, it means something is seriously wrong elsewhere.

Reply:
it's more about type check is throwing an error unless I put the if statement.

Reply:
Looks like I can enforce not to be null by doing`getDomain(node)!`